readme.txt

```diff
@@ -143,6 +143,11 @@ Since the extension is developing, there is currently not a specified way to tra
 
 == Upgrade Notice ==
 
+- 4.2.0 =
+
+Changes in the 4.2.0 branch implement future breaking changes to IndieAuth. Backward compatibility will be maintained for the foreseeable future, but clients are advised to update to the latest version of the standard
+to take advantage of the latest opportunities.
```

Comment:
I'm being super fickle, but it doesn't need a new line. ;P

Reply:
me too ;)